### PR TITLE
Simplify deploy by using commit hash as digest

### PR DESCRIPTION
**Pull request by corymcdonald**
#### Features

* Remove the `yarn build` step. 

We don't technically need this right now. This was originally added so that if a change is made on the public site but "yarn build" isn't run then the latest changes will still be deployed.

* Reverse engineer the Rails Webpacker and Sprockets pipeline so that we can use an arbitrary digest.

### CODE REVIEW

File: public/service-worker.js [hunk 1]
```diff
@@ -35,5 +35,5 @@ workbox.precaching.precacheAndRoute(self.__precacheManifest, {});
 
 workbox.routing.registerNavigationRoute(workbox.precaching.getCacheKeyForURL("/index.html"), {
   
-  blacklist: [/^\/_/,/\/[^/?]+\.[^/]+$/],
+  blacklist: [/^\/_/,/\/[^\/?]+\.[^\/]+$/],
```
Comment (corymcdonald) [inline public/service-worker.js:38]: Not sure why this change, but it happened when I ran `yarn build` before deleting the entry in `package.json`.

**Review by evq — APPROVED** [review]
